client/js/socket-events/init.js

```diff
@@ -163,7 +163,16 @@ function mergeChannelData(oldChannels, newChannels) {
 			// Server sends an empty users array, client requests it whenever needed
 			if (key === "users") {
 				if (channel.type === "channel") {
-					channel.usersOutdated = true;
```

Comment:
This should have been `currentChannel` instead of `channel`.

This line effectively did not because the `usersOutdated` prop was not copied into the existing object.